packages/stream_chat_flutter/lib/src/message_input.dart

```diff
@@ -495,6 +495,12 @@ class MessageInputState extends State<MessageInput> {
       child: Center(
         child: Container(
           clipBehavior: Clip.antiAlias,
+          margin: (widget.sendButtonLocation == SendButtonLocation.inside
```

Comment:
nitpick: Consider extracting logic to a variable at the top of our build function. It's a nice way to keep the build method clean 😉